packages/components/src/Tag/subcomponents/LiveIcon/LiveIcon.tsx

```diff
@@ -0,0 +1,67 @@
+// This file is autogenerated by wrapSVGs.ts
+// Changes to this file will be overwritten
+
+import React from 'react'
+import { SVG } from '~components/Icon/subcomponents/SVG'
+import type { IconProps } from '~components/Icon/types'
+import styles from './LiveIcon.module.css'
+
+const LiveIconSvg = (props: IconProps): JSX.Element => (
+  <SVG
+    // add the attributes that do not differ between your usages eg. role
+    {...props}
+  >
+    <path
+      stroke="currentColor"
+      d="M3.266 12.733c-2.6-2.6-2.6-6.866 0-9.466M5.2 10.8c-1.534-1.533-1.534-4.067 0-5.667"
+      strokeWidth="1.5"
+      strokeLinejoin="round"
+      strokeLinecap="round"
+    />
+    <path fill="currentColor" d="M8 10a2 2 0 1 0 0-4 2 2 0 0 0 0 4" />
+    <path
+      stroke="currentColor"
+      d="M10.8 5.2c1.533 1.533 1.533 4.067 0 5.667M12.733 3.267c2.6 2.6 2.6 6.8 0 9.4"
+      strokeWidth="1.5"
+      strokeLinejoin="round"
+      strokeLinecap="round"
+    />
+  </SVG>
+)
+
+export const LiveIcon = (): JSX.Element => (
```

Comment:
I'm surprised linting hasn't called you out for it 🤔 Add a displayName to the components - these assist with debugging and Storybook snippets